### PR TITLE
feelppbook request

### PR DESCRIPTION
**Pull request by bachir151**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,4 +43,4 @@ You can contribute to improve this documentation on
 
 ---
 
-{% include "SUMMARY.md" %}
```
Comment (prudhomm) [inline README.md:46]: why remove the Summary here ?

File: Tutorial/Keywords.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+Keywords
```
Comment (prudhomm) [inline Tutorial/Keywords.md]: Why did you copy this file ? it is in QuickReference ? we do not want to maintain two files which are the same
what is the idea here ?

**Comment by prudhomm** [comment]
@bachir151 @wkyoshe @vhuber 

there is `compiling.md` and `Tutorial/compiling.md` what is the rationale ?

The Tutorial is about Feel++ not about the prerequisites such as 
- compiling c++ code
- handling  libraries/tools used by feel++

this should probably go in a separate chapter e.g. `Prerequisites for Feel++`

**Comment by vhuber** [comment]
@wkyoshe @bachir151 @prudhomm I'm afraid there is no presentation of the CMakeLists.txt we have - for example - in the various research/project directory that is able to exploit the system installed Feel++ library if available.

**Pull request merged**